package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "notes",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.0.0-beta.2",
+    "json-server": "^0.14.2",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.1.3"
+  },
+  "scripts": {
+    "lint": "tslint --project ./tsconfig.json",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "json-server": "json-server --watch db.json --port 1111"
```

Comment:
Edit your README to include an obvious way to start the project together with backend.
You could also include a script to run BE and FE in parallel (another lib needed for this).

Reply:
thank you for advice! added such script